### DISCUSSION TITLE
Why Next.js auto injected GA(google analysis) code after project deployed with Vercel, and how to remove it?

### DISCUSSION THREAD

**Discussion by xgqfrms** [vercel/next.js#43647] [Help]
# Why Next.js auto injected GA(google analysis) code after project deployed with Vercel, and how to remove it?

> remove google analysis 

https://nextjs-ssr.xgqfrms.xyz/react_v18

![image](https://user-images.githubusercontent.com/7291672/205347209-2e270708-87dc-4923-b96b-a9b5778ed7c2.png)

![image](https://user-images.githubusercontent.com/7291672/205347339-ba27ff46-a826-467a-aaad-72190f8d8cce.png)

https://nextjs-ssr.xgqfrms.xyz/cdn-cgi/apps/head/rRAjM--Jr4OkvJzu1MIaJ4R9yCA.js

```js
{
  (function(i,s,o,g,r,a,m){
    i['GoogleAnalyticsObject']=r;
    i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)
    },
    i[r].l=1*new Date();
    a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];
    a.async=1;
    a.src=g;
    m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
};
if(CloudflareApps.matchPage(CloudflareApps.installs['2Jwcrmez-Neq'].URLPatterns)){
  (function(){
    var options=CloudflareApps.installs['2Jwcrmez-Neq'].options
    if(!options.id)return
    window.ga('create',options.id,'auto')
    window.ga('send','pageview')
    // ...
  })
}

```

![image](https://user-images.githubusercontent.com/7291672/205349193-bd755ce9-939f-4fa1-b532-bfc2ef819e26.png)

**Comment by wwwdepot** [comment]
You're using Cloudflare & using their apps. You need to go to your Cloudflare Dashboard disable and/or remove related apps for your domain.
